src/browser/keyboard.js

```diff
@@ -299,7 +299,7 @@ function KeyboardAdapter(bus)
         if(e.target)
         {
             // className shouldn't be hardcoded here
-            return e.target.className === "phone_keyboard" ||
+            return e.target.className.includes("phone_keyboard") ||
```

Comment:
```suggestion
            return e.target.classList.includes("phone_keyboard") ||
```

Reply:
```suggestion
            return e.target.classList.contains("phone_keyboard") ||
```
classList.includes doesn't exist